app/services/clients/ledger_client.js

```diff
@@ -8,6 +8,44 @@ const defaultOptions = {
   service: 'ledger'
 }
 
+const legacyConnectorTransactionParity = (transaction) => {
+  if (transaction.refund_summary && transaction.refund_summary.amount_refunded) {
+    transaction.refund_summary.amount_submitted = transaction.refund_summary.amount_refunded
+  }
+  transaction.charge_id = transaction.transaction_id
+  return transaction
+}
+
+const transaction = function transaction (id, gatewayAccountId, options = {}) {
+  const configuration = Object.assign({
+    url: `/v1/transaction/${id}`,
+    qs: { account_id: gatewayAccountId },
+    description: 'Get individual transaction details',
+    transform: legacyConnectorTransactionParity
+  }, defaultOptions, options)
+  return baseClient.get(configuration)
+}
+
+const events = function events (transactionId, gatewayAccountId, options = {}) {
+  const legacyConnectorEventsParity = (transactionEvents) => {
+    transactionEvents.events = transactionEvents.events.map(event => {
+      return Object.assign({
+        type: event.resource_type.toLowerCase(),
+        updated: event.timestamp,
+        ...(event.data.refunded_by) && { submitted_by: event.data.refunded_by }
+      }, event)
+    })
+    return transactionEvents
+  }
```

Comment:
[minor] for `events` the `legacyConnector...` is internal function, but for `transaction` it's not. Would it be possible to be more consistent?

Reply:
`transaction` will be used to map both an individual transaction and the transactions picked up in transaction search -- events will only ever be transformed within this one end point. I see how that's not clear until the second end point has been added